public/electron.js

```diff
@@ -46,18 +46,16 @@ function getOpenedWindow(url,options) {
 
   options.webPreferences.affinity = 'main-window';
 
-  if(url.includes('about:blank')){
-    /*
+  if(url.includes('trezor')){
     session.defaultSession.webRequest.onBeforeSendHeaders(filter, (details, callback) => {
-      details.requestHeaders['Origin'] = 'https://electron.trezor.io';
+      details.requestHeaders['Origin'] = 'https://connect.trezor.io';
       callback({cancel: false, requestHeaders: details.requestHeaders});
     });
-    */
   }
 
-  if(url.includes('wallet.portis') || url.includes('about:blank') || url.includes('app.tor.us')){
+  if(url.includes('wallet.portis') || url.includes('trezor') || url.includes('app.tor.us')){
```

Comment:
perhaps rewrite it as: `['wallet.portis', 'wallet.trezor', 'app.tor.us'].includes(url)`

Reply:
It's not the same though. For example : 
```
const domains = ['app.tor.us','wallet.portis']
domains.includes('www.wallet.portis')
**false**
```